src/test/java/com/shapesecurity/shift/es2016/test262/Test262.java

```diff
@@ -0,0 +1,385 @@
+package com.shapesecurity.shift.es2016.test262;
+
+import com.shapesecurity.functional.data.ImmutableList;
+import com.shapesecurity.shift.es2016.ast.Program;
+import com.shapesecurity.shift.es2016.ast.Script;
+import com.shapesecurity.shift.es2016.parser.EarlyError;
+import com.shapesecurity.shift.es2016.parser.EarlyErrorChecker;
+import com.shapesecurity.shift.es2016.parser.JsError;
+import com.shapesecurity.shift.es2016.parser.Parser;
+import org.junit.Assert;
+import org.junit.Test;
+import org.yaml.snakeyaml.Yaml;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class Test262 {
+
+	private static final Yaml yamlParser = new Yaml();
+
+	private static final HashSet<String> xfailParse = new HashSet<>(Arrays.asList(
+			// shift-java#198
+			"src/test/resources/test262/test/annexB/language/statements/for-in/nonstrict-initializer.js",
```

Comment:
Can you dedupe the test path? These should all start with `src/test/resources/test262/test` so extract that. This will make it easier in case we want to change the location of our test262 submodule or something.